src/main/resources/assets/translation/en.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
It would be great as well to also translate the text written while playing the game.